### PR TITLE
chore: add documentations notice with the current type of strings and how to scale them with decimal places

### PR DESCRIPTION
**Pull request by jeremyletang**

### CODE REVIEW

**Review by AnExsomnis — DISMISSED** [review]

File: protos/sources/vega/vega.proto [hunk 1]
```diff
@@ -1208,14 +1265,19 @@ message Node {
...
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
   string staked_by_delegates = 8;
   // Total amount staked on node
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
   string staked_total = 9;
   // Max amount of (wanted) stake, is this a network param or a node param
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
   string max_intended_stake = 10;
   // Amount of stake on the next epoch
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1280]:
```suggestion
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places for the token.
  string staked_by_operator = 7;
  // The amount of stake that has been delegated by token holders
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places for the token.
  string staked_by_delegates = 8;
  // Total amount staked on node
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places for the token.
  string staked_total = 9;
  // Max amount of (wanted) stake
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
  string max_intended_stake = 10;
  // Amount of stake on the next epoch
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places of the token.
```

File: protos/sources/vega/vega.proto [hunk 2]
```diff
@@ -1250,6 +1312,7 @@ message NodeSet {
 
 message NodeData {
   // Total staked amount across all nodes
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1315]:
```suggestion
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places of the token.
```

File: protos/sources/vega/vega.proto [hunk 3]
```diff
@@ -1271,6 +1334,7 @@ message Delegation {
   // Node ID
   string node_id = 2;
   // Amount delegated
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1337]:
```suggestion
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places of the token.
```

File: protos/sources/vega/vega.proto [hunk 4]
```diff
@@ -1281,6 +1345,7 @@ message Reward {
   string asset_id = 1;
   string party_id = 2;
   uint64 epoch = 3;
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1348]:
```suggestion
  // Amount paid as a reward
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places of the reward asset.
```

Comment (candida-d) [reply protos/sources/vega/vega.proto:1348]: @jeremyletang please confirm I've got this right?

File: protos/sources/vega/vega.proto [hunk 5]
```diff
@@ -1293,6 +1358,7 @@ message RewardSummary {
   string asset_id = 1;
   string party_id = 2;
   // Total amount of rewards for the asset
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1361]:
```suggestion
  // Total amount of rewards paid in the asset
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset's decimal places.
```

File: protos/sources/vega/vega.proto [hunk 6]
```diff
@@ -1302,6 +1368,7 @@ message EpochRewardSummary {
   string asset_id = 2;
   string market_id = 3;
   string reward_type = 4;
+  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1371]:
```suggestion
 // Amount distributed
  // This field is an unsigned integer passed as a string and needs to be scaled using the asset decimal places.
```

**Review by AnExsomnis — DISMISSED** [review]
LGTM after Candida's comments, but @jeremyletang , there are 51 commits now, maybe you would want to squash?

**Comment by jeremyletang** [comment]
> LGTM after Candida's comments, but @jeremyletang , there are 51 commits now, maybe you would want to squash?

I did rebase and squash indeed @AnExsomnis, you can go and approve again if you like! Cheers.

**Review by candida-d — DISMISSED** [review]

**Review by AnExsomnis — DISMISSED** [review]

**Comment by jeremyletang** [comment]
@AnExsomnis because I'm an idiot I didn't regen the protos after the rebase... Would you mind approve again?

**Review by candida-d — APPROVED** [review]

**Review by AnExsomnis — APPROVED** [review]

**Pull request merged**